docs/reference/execute-methods.md

```diff
@@ -199,11 +199,65 @@ bundleId | string | yes | The bundle identifier of the application to be removed
 
 #### Returned Result
 
-Either `true` if the app was successfully uninstalled, otherwise `false`
+Either `true` if the application was successfully uninstalled, otherwise `false`
 
 ### mobile: launchApp
 
-Executes the given app on the device under test. If the app is already running then it would be activated. If the app is not installed or cannot be launched then an exception is thrown.
+Executes the given application on the device under test. If the application is already running then it would be activated.
+If the application is not installed or cannot be launched then an exception is thrown.
+
+It accepts `arguments` and `environment` to start an application with them.
+
+An example of the `arguments` usage is language and locale.
+XCTest lets you start an app process by specifing [Language and Locale IDs](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/LanguageandLocaleIDs/LanguageandLocaleIDs.html) via the process arguments with `-AppleLanguages` and `-AppleLocale`.
+The format is [Testing Specific Languages and Regions](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/TestingYourInternationalApp/TestingYourInternationalApp.html).
+Please terminate the application if the process was already running. The process should start with the arguments.
```

Comment:
> Please terminate the application if the process was already running. The process should start with the arguments.

This is redundant. launchApp would terminate the application anyway to reinstall it

Reply:
hm, do you mean `mobile:installApp` command? https://github.com/appium/WebDriverAgent/blob/3c8497be55ddad0524cfb02c6db94a9c24e5076a/WebDriverAgentLib/Routing/FBSession.m#L206
My local check also needed to terminate the bundle id if the bundle id was already running

Reply:
yes, I've mixed it with installApp. Please ignore.